### PR TITLE
Commit description

### PR DESCRIPTION
**Pull request by DemianParkhomenko**
....

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const PORT = 8000
 http
   .createServer((req, res) => {
     res.writeHead(200, { 'Content-Type': 'text/plain' })
-    res.end('Hello World!')
+    res.end('Hello World!!!!!!!!!')
```
Comment (NazarMykolenko) [inline index.js:8]: Hello world

**Review by NazarMykolenko — APPROVED** [review]
Goggins

**Review by DemianParkhomenko — COMMENTED** [review]
LGTM

File: index.js (hunk 1 above)
Comment (masalovd) [inline index.js:8]: Hello, Naz

**Review by masalovd — APPROVED** [review]

**Pull request merged**